static/fetch.js

```diff
@@ -1,31 +1,179 @@
 const RELOAD_TIME = 3000; 
-const PERSISTENCE_TIME = 180000;
+const PERSISTENCE_TIME_COMPLETE = 180000;
+const PERSISTENCE_TIME_PENDING = 300000;
+const PERSISTENCE_TIME_ERROR = 300000;
+
+const PrintJob = {
+    PARSE(code) {
+        for (let key in PrintJob) {
+            if (PrintJob[key].CODE === code) return PrintJob[key]
+        }
+    },
+    COMPLETED: {
+        CODE: 0,
+        ATTR: 'completed',
+        MSG: 'SUCCESS'
+    },
+    PENDING: {
+        CODE: 1,
+        ATTR: 'pending',
+        MSG: 'PENDING'
+    },
+    FILE_ERROR: {
+        CODE: 2,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    },
+    QUOTA_LIMIT: {
+        CODE: 3,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    },
+    JOB_ERROR: {
+        CODE: 4,
+        ATTR: 'error',
+        MSG: 'FAILURE'
+    }
+}
+
+
+function _element(tag, classList, attributes, innerText){
+    let temp = document.createElement(tag);
+    if (classList && !(classList instanceof Array) && typeof classList === 'string') classList = [classList];
+    else classList = [];
+    if (attributes && attributes instanceof Array) {
+        let temp = {};
+        attributes.forEach(item => {
+            temp[item] = true;
+        });
+        attributes = temp;
+    } else if (attributes && typeof attributes === 'string') {
+        attributes = {[attributes]: true}
+    } else {
+        attributes = {};
+    }
+    classList.forEach(className => temp.classList.add(className));
+    for (let key in attributes) {
+        temp.setAttribute(key, attributes[key] && typeof attributes[key] === 'boolean' ? '' : String(attributes[key]));
```

Comment:
can just do the `setAttribute` in the if statements above and avoid casing out the same thing twice?